### PR TITLE
Import router in oauth js

### PR DESCRIPTION
**Pull request by rullzer**
Again a bit more cleanup

### CODE REVIEW

File: apps/oauth2/src/App.vue [hunk 1]
```diff
@@ -55,7 +55,8 @@
 
 <script>
 import Axios from 'nextcloud-axios'
-import OAuthItem from './components/OAuthItem';
+import OAuthItem from './components/OAuthItem'
+import { generateUrl} from 'nextcloud-router'
```
Comment (skjnldsv) [inline apps/oauth2/src/App.vue]:
```suggestion
import { generateUrl } from 'nextcloud-router'
```

**Review by ChristophWurst — APPROVED** [review]
:+1: :)

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
Tests fails :)

File: apps/oauth2/src/App.vue [hunk 2]
```diff
@@ -75,6 +75,7 @@
 <script>
 import axios from 'nextcloud-axios'
 import OAuthItem from './components/OAuthItem'
+import { generateUrl } from 'nextcloud-router'
```
Comment (ChristophWurst) [inline apps/oauth2/src/App.vue]: nextcloud-router is so September 2019

https://www.npmjs.com/package/@nextcloud/router it is

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**